pandas/tests/frame/test_apply.py

```diff
@@ -59,66 +58,66 @@ def test_apply_mixed_datetimelike(self):
         # mixed datetimelike
         # GH 7778
         df = DataFrame({'A': date_range('20130101', periods=3),
-                        'B': pd.to_timedelta(np.arange(3), unit='s')})
+                        'B': to_timedelta(np.arange(3), unit='s')})
         result = df.apply(lambda x: x, axis=1)
-        assert_frame_equal(result, df)
+        tm.assert_frame_equal(result, df)
```

Comment:
Same comment as before - I like the explicit namespacing and agree it would be good to make that consistent in this module, but better done in a separate, contained PR

Reply:
As above as well:
> IMO these are trivial changes that don't get done otherwise, [...] However, I separated the commits between fixturizing and import-cleaning, so you can review the separate commits (almost) as separate PRs.